### ISSUE TITLE
Create holdout data set based on generate_data.stan

### ISSUE THREAD

**Issue by dgmiller** [marcdotson/conjoint-ensembles#18]
Increase the number of tasks (e.g. from 10 to 20) then split the data set into two halves. The splitting might just be done in R or Python so routines should be written for both. Files in each language should read in generate_data.stan, split into training and testing sets, fit each treatment model on the training set, stack the posterior distributions, and determine out-of-sample predictive fit on the test set.

A separate issue will be created to code up a demonstration and comparison between the predictive fit of each model individually vs the stacking ensemble.
